packaging/rules

```diff
@@ -6,8 +6,8 @@ RPMBUILD=rpmbuild --define "_topdir `pwd`/packaging" --define "cccver $(VERSION)
 help:
 	@echo "Targets:"
 	@echo "    tgz:   Build a cellcc-v*.tar.gz source tarball"
-	@echo "    srpm7: Build an srpm for EL7"
```

Comment:
I prefer to keep separate spec files around to avoid complicated %if checks inside the spec, but if it really doesn't need any changes for rhel8 and modern fedora, then that's perfectly fine.

even with separate specfiles, I suppose if we wanted, we could just detect what platform we're on and pick a spec file from that.